settings.gradle

```diff
@@ -2,14 +2,11 @@ pluginManagement {
     repositories {
         mavenCentral()
         gradlePluginPortal()
-        maven {
-            url 'https://repo.futit.cloud/repository/maven-public-snapshots'
-        }
         maven {
             url 'https://repo.futit.cloud/repository/maven-public-releases'
         }
         maven {
-            url 'https://repository-cdn.liferay.com/nexus/content/repositories/liferay-public-releases'
+            url 'https://repo.futit.cloud/repository/maven-public-snapshots'
```

Comment:
why snapshots ?

Reply:
We still don't have a stable release for EtendoRX's jar version